### PR TITLE
Add Dungeonchain paths to IBC registry

### PR DESCRIPTION
**Pull request by ApeironNodes**
Added 

dungeonchain-osmosis
dungeonchain-cosmoshub

Channels checked and working.

### CODE REVIEW

**Review by Cordtus — CHANGES_REQUESTED** [review]
The chain names must be in alphabetical order.

**Comment by ApeironNodes** [comment]
> The chain names must be in alphabetical order.

Resolved, thanks for the heads up!

**Review by ApeironNodes — COMMENTED** [review]
Resolved

File: _IBC/cosmoshub-dungeonchain.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "../ibc_data.schema.json",
+    "chain_1": {
+      "chain_name": "cosmoshub",
+      "client_id": "07-tendermint-4",
+      "connection_id": "connection-1057"
+    },
+    "chain_2": {
+      "chain_name": "dungeonchain",
```
Comment (JeremyParish69) [inline _IBC/cosmoshub-dungeonchain.json:9]: checks failed because 'dungeonchain' is not the chain_name--it's just 'dungeon'

**Comment by ApeironNodes** [comment]
@JeremyParish69

**Pull request merged**